src/guides/v2.2/rest/tutorials/orders/order-add-items.md

```diff
@@ -421,6 +421,47 @@ For this example, we'll configure the Sprite Yoga Companion Kit as follows:
 
 {% endcollapsible %}
 
+This calls are performed on behalf of a guest-customer, and there is no need of token.
+
+### Add a simple product to a cart {#add-simple}
+
+**Endpoint:**
+
+`POST <host>/rest/<store_code>/guest-carts/<cartId>/items`
+
+Cart Id is the quoteId recieved on quote creation.
```

Comment:
Could I kindly ask you to make the [quote creation] as a link to the corresponding topic? Thank you

Reply:
Updated as per your comments

Reply:
Updated as per your comments